src/devices/frient.ts

```diff
@@ -1,18 +1,61 @@
 import {Definition} from '../lib/types';
 import * as exposes from '../lib/exposes';
 import fz from '../converters/fromZigbee';
+import tz from '../converters/toZigbee';
 import * as reporting from '../lib/reporting';
+import * as constants from '../lib/constants';
+import * as utils from '../lib/utils';
+
 const e = exposes.presets;
 
+const manufacturerOptions = {manufacturerCode: 0x1015};
+
+const frient = {
+    fz: {
+        pulse_configuration: {
+            cluster: 'seMetering',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                const result = {};
+                if (msg.data.hasOwnProperty('develcoPulseConfiguration')) {
+                    result[utils.postfixWithEndpointName('pulse_configuration', msg, model, meta)] =
+                        msg.data['develcoPulseConfiguration'];
+                }
+
+                return result;
+            },
+        },
+    tz: {
+        pulse_configuration: {
+            key: ['pulse_configuration'],
+            convertSet: async (entity, key, value, meta) => {
+                await entity.write('seMetering', {'develcoPulseConfiguration': value}, manufacturerOptions);
+                return {readAfterWriteTime: 200, state: {'pulse_configuration': value}};
+            },
+            convertGet: async (entity, key, meta) => {
+                await entity.read('seMetering', ['develcoPulseConfiguration'], manufacturerOptions);
+            },
+        },
+    }
+};
+
+
+
 const definitions: Definition[] = [
     {
         zigbeeModel: ['EMIZB-141'],
         model: 'EMIZB-141',
         vendor: 'frient',
         description: 'Smart powermeter Zigbee bridge',
-        fromZigbee: [fz.metering, fz.battery],
-        toZigbee: [],
-        exposes: [e.battery(), e.power(), e.energy()],
+        fromZigbee: [fz.metering, fz.battery, frient.fz.pulse_configuration],
+        toZigbee: [frient.tz.pulse_configuration],
+        exposes: [
```

Comment:
Could you use the modernExtend for this instead? Example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/68eee46b7c3fbd79a08d90124c94c368de15041f/src/devices/xiaomi.ts#L987